src/pages/elements/logos/rebus.mdx

```diff
@@ -51,7 +51,7 @@ our offerings.
 
 ## IBM Rebus
 
-Designed by Paul Rand for his Eye-Bee-M poster (1981) in support of IBM’s motto, THINK. The Rebus used pictures to represent letters which are an iconic part of IBM’s visual history. Today we still the use the Rebus which has since been adjusted to meet the requirements of differerent mediums, scales and production environments. Like the 8-bar, the Rebus comes in both positive and negative versions. The Rebus can be used in special applications such as merchandise, badges and events but should not be used in place of the 8-Bar as an official brand communications sign off.
+Designed by Paul Rand for his Eye-Bee-M poster (1981) in support of IBM’s motto, THINK. The Rebus used pictures to represent letters which are an iconic part of IBM’s visual history. Today we still the use the Rebus which has since been adjusted to meet the requirements of different mediums, scales and production environments. Like the 8-bar, the Rebus comes in both positive and negative versions. The Rebus can be used in special applications such as merchandise, badges and events but should not be used in place of the 8-Bar as an official brand communications sign off.
```

Comment:
```suggestion
Designed by Paul Rand for his Eye-Bee-M poster (1981) in support of IBM’s motto, THINK. The Rebus used pictures to represent letters which are an iconic part of IBM’s visual history. Today we still use the Rebus which has since been adjusted to meet the requirements of different mediums, scales and production environments. Like the 8-bar, the Rebus comes in both positive and negative versions. The Rebus can be used in special applications such as merchandise, badges and events but should not be used in place of the 8-Bar as an official brand communications sign off.
```

Reply:
It's hard to see but there is an unnecessary "the" in this sentence: "Today we still the use the Rebus which has since been adjusted to meet the requirements..."  that I removed in the suggestion.